pkg/apis/shipper/v1alpha1/types.go

```diff
@@ -8,7 +8,8 @@ import (
 )
 
 const (
-	ShipperNamespace = "shipper-system"
+	ShipperNamespace            = "shipper-system"
+	GlobalRolloutBlockNamespace = "global-rollout-blocks"
```

Comment:
Minor suggestion: rename it to `rollout-block-global`. In this case if we introduce alternative namespaces to this one, the autocomplete-aware clients will still work and suggest all these namespaces.